TOC frontend styles

carl-alberto opened the pull request:
Added table of content sidebar styles https://cloudup.com/ct4-XtLKMuO

cralberto11 committed:
Added table of content sidebar
https://cloudup.com/ct4-XtLKMuO

ntwb commented on themes/helphub/style.css:
```diff
@@ -1249,3 +1249,58 @@ h2.helphub-footerarea-title {
+}
+
+.single-post h1.entry-title {
+	font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, sans-serif;
```
`"HelveticaNeue-Light"` shouldn't be wrapped in quotes

ntwb commented on themes/helphub/style.css:
```diff
@@ -1249,3 +1249,58 @@ h2.helphub-footerarea-title {
+}
+
+.table-of-contents ul {
+	margin-left: .2em;
```
This should be `0.2em`

ntwb reviewed (changes requested):
Also use tabs for indentation please

cralberto11 committed:
Fixed build errors and recommendations by @ntwb

 1267:16  ✖  Unexpected quotes around                   font-family-name-quotes
>>              "HelveticaNeue-Light"                                             
>>  1282:15  ✖  Expected a leading zero                    number-leading-zero    
>>  1286:5   ✖  Expected indentation of 1 tab              indentation            
>>  1288:5   ✖  Expected indentation of 1 tab              indentation            
>>  1292:5   ✖  Expected indentation of 1 tab              indentation            
>>  1294:5   ✖  Expected indentation of 1 tab              indentation            
>>  1298:5   ✖  Expected indentation of 1 tab              indentation            
>>  1300:5   ✖  Expected indentation of 1 tab              indentation

ntwb commented on themes/helphub/style.css:
```diff
@@ -1249,3 +1249,58 @@ h2.helphub-footerarea-title {
+}
+
+.single-post h1.entry-title {
+	font-family: HelveticaNeue-Light, Helvetica Neue Light, Helvetica Neue, Helvetica, Arial, sans-serif;
```
>>  1267:36  ✖  Expected quotes around "Helvetica Neue Light"     font-family-name-quotes                                        
>>  1267:36  ✖  Expected quotes around "Helvetica Neue"    font-family-name-quotes

You removed quotes from these two, they still require quotes

ntwb replied:
From the stylelint rule docs"

> https://stylelint.io/user-guide/rules/font-family-name-quotes/
> _Quotes are recommended in the spec with "font family names that contain white space, digits, or punctuation characters other than hyphens"._

carl-alberto replied:
Thanks for the correction @ntwb, updated

cralberto11 committed:
Added quotes for font family names that has whitespaces
